rasa/importers/multi_project.py

```diff
@@ -3,6 +3,8 @@
 from typing import Text, Set, Dict, Optional, List, Union, Any
 import os
 
+from async_lru import alru_cache
```

Comment:
I would avoid adding this new library because
- it doesn't seem well maintained (only three releases, 
- we can implement a similar logic without too much effort by using class attributes

e.g. 
```python

async def get_domain(self) -> Domain:
    if self._merged_domain:
        return self._merged_domain
    else:
        domains = [Domain.load(path) for path in self._domain_paths]
        ...
```

Reply:
OK I followed your advice and removed the async_lru dependency completely, with something along your lines to cache domain & stories & nlu_data.